src/cfg.py

```diff
@@ -163,22 +163,22 @@
     },
     completion_cfg={
         "completion_kwargs": {
-            "model": "gpt-3.5-turbo-0613",
+            "model": "gpt-4o-mini",
             "stream": True,
             "temperature": 0,
         },
         "client_kwargs": client_kwargs,
     },
     tokenizer_cfg={
-        "model_name": "gpt-3.5-turbo-0613",
+        "model_name": "gpt-4",
```

Comment:
note it's equivalent to 4o-mini, but the `tiktoken` version we depend on (because of `buster-doctalk`) doesn't list it yet